index.js

```diff
@@ -0,0 +1,114 @@
+class Bank {
+    #accounts
+    #loggedIn
+    #date
+    #transactions
+    constructor(date, accounts) {
+        this.#date = date
+        this.#accounts = accounts
+        this.#loggedIn = false
+    }
+    date(date) {
+        if (typeof date !== "string") {
+            throw "date must be a string"
+        }
+        this.#date = date
+    }
+    get dateValue() {
+        return this.#date
+    }
+    get loggedIn() {
+        return this.#loggedIn
```

Comment:
Try to name booleans like it's answering a yes / no question, e.g. `isLoggedIn` or `hasSomething`